Packages/com.unity.inputsystem/InputSystem/Actions/InputActionParameters.cs

```diff
@@ -0,0 +1,805 @@
+        /// <example>
+        /// <code>
+        /// // Apply a parameter override only to action1.
+        /// map.ApplyBindingOverride("clamp:min", 0.25f, new InputBinding { action = action1.name });
+        ///
+        /// // Apply a parameter override only to a specific binding path.
+        /// map.ApplyBindingOverride("clamp:min", 0.4f, new InputBinding { path = "&lt;Gamepad&gt;/leftTrigger" });
+        /// </code>
+        /// </example>
+        ///
+        /// If there are multiple overrides that could be applied to a specific parameter, the most specific override
+        /// is chosen. For example, if both an override for "duration" and an override for "duration" specifically on bindings
+        /// to the "look" action exists, then the latter is used for all bindings to the "look" action and the former is used
+        /// for all other bindings. For parameter overrides of equal specificity, the behavior is undecided.
+        ///
+        /// Note that parameter overrides stay in place on the map. Like binding overrides, however, they are not
+        /// automatically persisted and thus need to be reapplied when actions are loaded from assets. This will, however, be applied
+        /// automatically to bindings added to the action in the future as well as whenever bindings are resolved to controls.
+        /// </remarks>
+        /// <exception cref="ArgumentNullException"><paramref name="actionMap"/> is <c>null</c> -or- <paramref name="name"/> is <c>null</c>
+        /// or empty.</exception>
+        /// <seealso cref="GetParameterValue(InputAction,string,InputBinding)"/>
+        public static void ApplyParameterOverride(this InputActionMap actionMap, string name, PrimitiveValue value, InputBinding bindingMask = default)
+        {
+            if (actionMap == null)
+                throw new ArgumentNullException(nameof(actionMap));
+            if (string.IsNullOrEmpty(name))
+                throw new ArgumentNullException(nameof(name));
+
+            actionMap.ResolveBindingsIfNecessary();
+
+            ApplyParameterOverride(actionMap.m_State, actionMap.m_MapIndexInState,
+                ref actionMap.m_ParameterOverrides, ref actionMap.m_ParameterOverridesCount,
+                name, value, bindingMask);
+        }
+
+        /// <summary>
+        /// Set the value of the given parameter on the <see cref="InputBindingComposite"/>, <see cref="IInputInteraction"/>,
+        /// and <see cref="InputProcessor"/> objects found on the <see cref="InputActionMap.bindings"/> of each of the <see cref="InputActionAsset.actionMaps"/>
+        /// in <paramref name="asset"/>.
+        /// </summary>
+        /// <param name="asset">An <c>.inputactions</c> asset on whose <see cref="InputActionMap.bindings"/> to look for objects to set
+        /// the parameter value on.</param>
+        /// <param name="name">Name of the parameter to get the value of. Case-insensitive. This can either be just the name of the
```

Comment:
We have a lot of redundant docs (more maintenence) between these methods. Do you know if there is any way to cross-reference or document method groups with the doc generator (docfx)?

Reply:
ok